src/main/java/frc/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -116,11 +187,51 @@ public DrivebaseSubsystem() {
 
     swerveModules = // modules are always initialized and passed in this order
         new SwerveModule[] {frontRightModule, frontLeftModule, backLeftModule, backRightModule};
+
+    odometryXEntry = createEntry("X", 0);
+    odometryYEntry = createEntry("Y", 1);
+    odometryAngleEntry = createEntry("Angle", 2);
+
+    // This is an example of how you can use a lambda instead of the NetworkTableEntry method
+    tab.addNumber(
+        "Rotation Speed rad/s",
+        () -> {
+          ChassisSpeeds speeds;
+          speeds = this.chassisSpeeds;
+          if (speeds == null) return 0.0;
+
+          return speeds.omegaRadiansPerSecond;
+        });
+  }
+
+  /** Return the current pose estimation of the robot */
+  public Pose2d getPose() {
+    return robotPose;
+  }
+
+  /** Return the kinematics object, for constructing a trajectory */
+  public SwerveDriveKinematics getKinematics() {
+    return kinematics;
+  }
+
+  /**
+   * Tells the subsystem to drive, and puts the state machine in drive mode
+   *
+   * @param chassisSpeeds the speed of the chassis desired
+   */
+  public void drive(ChassisSpeeds chassisSpeeds) {
+    this.mode = Modes.DRIVE;
+    this.chassisSpeeds = chassisSpeeds;
+  }
+
+  public void resetGyroAngle(Rotation2d angle) {
```

Comment:
I have pretty mixed feelings about taking a parameter we do not use... Having an overloaded function in the future, and this function with no parameters now seems like a better path imo.

Reply:
Agreed. This was initially written to set the angle adjustment using the àngle param but is no longer doing that. I'll change to update